routific.gemspec

```diff
@@ -1,6 +1,6 @@
 Gem::Specification.new do |s|
   s.name              = 'routific'
-  s.version           = '1.1.2'
+  s.version           = '1.7.0'
   s.date              = '2017-06-26'
```

Comment:
Let's also change the `date` to `2017-09-21`.

Can you also remove `s.authors` and update `s.email` to `support@routific.com`?

Reply:
Sure!

Reply:
Hmm, apparently `authors may not be empty`